packages/oui-list/_mixins.less

```diff
@@ -188,203 +97,4 @@
       font-weight: @oui-font-semibold;
     }
   }
-
-  .list-separated (
-    @selector,
-    @separator-color: @oui-list-separated-separator-color,
-    @separator-size: @oui-list-separated-separator-size
-  ) {
-    .@{selector}__item {
-      border-bottom: @separator-size solid @separator-color;
-
-      &:last-child {
-        border-bottom: 0 none;
-      }
-    }
-  }
-
-  .list-steps (
```

Comment:
It doesn't cause breaking changes? only a move to packages/oui-list/list.less?

Reply:
Yes, I just move the useless mixins in the main file :)